pkg/agent/sysadvisor/plugin/qosaware/resource/cpu/advisor.go

```diff
@@ -306,7 +302,7 @@ func (cra *cpuResourceAdvisor) assignContainersToRegions() error {
 func (cra *cpuResourceAdvisor) assignToRegions(ci *types.ContainerInfo) ([]region.QoSRegion, error) {
 	if ci.QoSLevel == consts.PodAnnotationQoSLevelSharedCores {
 		// Assign shared cores container. Focus on pool.
-		if regions, ok := cra.getPoolRegions(ci.OwnerPoolName); ok {
+		if regions, ok := cra.getContainerRegions(ci); ok {
```

Comment:
Should not get region by container's region for shared cores container, because it's not possible to distinguish shared cores containers of different share pool by this way

Reply:
Still need to maintain pool region map in metacache, and assign shared cores container by its pool.